src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/utils.tsx

```diff
@@ -70,10 +70,18 @@ function getMethodTypeSelectorFieldLabel(labelType: METHOD_TYPE): string {
       return '';
   }
 }
+enum EVENT_ID_FIELD_STATUS {
+  LOADING = 'loading',
+  INVALID = 'invalid',
+  NOT_FOUND = 'not_found',
+  LOADED = 'loaded',
+  ERROR = 'error',
+}
```

Comment:
It looks like we generally don't use all caps for enums:

![image](https://user-images.githubusercontent.com/79684/80510499-a7ce3a00-892f-11ea-9bd4-618590215565.png)